src/app/static/src/app/components/outputTable/Filters.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
This component is going to contain both the presets drop down and also (probably) a generic `Filters` component, isn't it (from `components/plots/Filters`)? So it might be a bit confusing for this to just be called `Filters` too? Maybe `tableFilters` or `tableSelections`?

Reply:
I like table filters! lets do that actually